scripts/makefile/stamp.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
why is this called stamp.sh?

Reply:
stamping a template kind of thing. I will change/clarify